docs/apache-airflow-providers-amazon/operators/eks.rst

```diff
@@ -0,0 +1,265 @@
+
+In the following code we retrieve details for a given Amazon EKS Cluster.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_cluster.py
+    :language: python
+    :start-after: [START howto_operator_eks_describe_cluster]
+    :end-before: [END howto_operator_eks_describe_cluster]
+
+
+.. _howto/operator:EKSDeleteClusterOperator:
+
+Deleting Amazon EKS Clusters
+----------------------------
+
+Defining tasks
+""""""""""""""
+
+In the following code we delete a given Amazon EKS Cluster.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_cluster.py
+    :language: python
+    :start-after: [START howto_operator_eks_delete_cluster]
+    :end-before: [END howto_operator_eks_delete_cluster]
+
+
+.. _howto/operator:EKSCreateNodegroupOperator:
+
+Creating Amazon EKS Managed NodeGroups
+--------------------------------------
+
+Purpose
+"""""""
+
+This example dag ``example_eks_create_nodegroup.py`` uses ``EKSCreateNodegroupOperator``
+to create an Amazon EKS Managed Nodegroup using an existing cluster, ``EKSListNodegroupsOperator``
+and ``EKSDescribeNodegroupOperator`` to verify creation, then ``EKSDeleteNodegroupOperator``
+to delete the nodegroup.
+
+Prerequisites
+"""""""""""""
+
+An AWS IAM role with the following permissions:
+
+  "ec2.amazon.aws.com" must be in the Trusted Relationships
+  "AmazonEC2ContainerRegistryReadOnly" IAM Policy must be attached
+  "AmazonEKSWorkerNodePolicy" IAM Policy must be attached
+
+Defining tasks
+""""""""""""""
+
+In the following code we create a new Amazon EKS Managed Nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_create_nodegroup]
+    :end-before: [END howto_operator_eks_create_nodegroup]
+
+
+.. _howto/operator:EKSListNodegroupsOperator:
+.. _howto/operator:EKSDescribeNodegroupOperator:
+
+Listing and Describing Amazon EKS Clusters
+-------------------------------------------
+
+Defining tasks
+""""""""""""""
+
+In the following code we retrieve details for a given Amazon EKS nodegroup.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_describe_nodegroup]
+    :end-before: [END howto_operator_eks_describe_nodegroup]
+
+
+In the following code we list all Amazon EKS Nodegroups in a given EKS Cluster.
+
+.. exampleinclude:: /../../airflow/providers/amazon/aws/example_dags/example_eks_create_nodegroup.py
+    :language: python
+    :start-after: [START howto_operator_eks_list_nodegroup]
+    :end-before: [END howto_operator_eks_list_nodegroup]
+
+
+.. _howto/operator:EKSDeleteNodegroupOperator:
+
+Deleting Amazon EKS Managed Nodegroups
+--------------------------------------
+
+Defining tasks
+""""""""""""""
+
+In the following code we delete an Amazon EKS nodegroup.
```

Comment:
Ideally, you should use the class name in the operator description. The class name should be usable as a link, so you must use a reference:
``:class:`~airfflow.providers.amazon.aws.operators.EKSDeleteNodegroupOperator`. ``

Reply:
Should be addressed with https://github.com/apache/airflow/pull/16571/commits/5743f3469c8257948b9b73e2a7b2bdafb2a1d832